Userland/Libraries/LibJS/Runtime/FunctionPrototype.cpp

```diff
@@ -131,13 +131,15 @@ JS_DEFINE_NATIVE_FUNCTION(FunctionPrototype::to_string)
         StringBuilder parameters_builder;
         auto first = true;
         for (auto& parameter : script_function.parameters()) {
-            if (!first)
-                parameters_builder.append(", ");
-            first = false;
-            parameters_builder.append(parameter.name);
-            if (parameter.default_value) {
-                // FIXME: See note below
-                parameters_builder.append(" = TODO");
+            if (auto* name_ptr = parameter.binding.get_pointer<FlyString>()) {
+                if (!first)
+                    parameters_builder.append(", ");
+                first = false;
+                parameters_builder.append(*name_ptr);
+                if (parameter.default_value) {
+                    // FIXME: See note below
+                    parameters_builder.append(" = TODO");
+                }
             }
```

Comment:
Let's add a FIXME for the binding pattern case, which will eventually need stringification as well